README.md

```diff
@@ -145,3 +145,7 @@ An example of an embedded citation object from Mendeley:
     }
 }
 ```
+
+# Licensing
+
+This repository is released under the MIT license.
```

Comment:
Consider linking to license file like:

```md
This repository is released under the [MIT license](LICENSE.txt).
```

Or even removing this section since most users will know that the LICENSE file is the relevant reference here.